### PR TITLE
Movie card

### PR DESCRIPTION
**Pull request by younessoudouh**
-create movie card component
-display movie card component

Screenshot (5)

### CODE REVIEW

**Comment by faouzioudouh** [comment]
Can you add a screenshot of the progress in the PR description?

File: src/Componnets/movie-card/MovieCard.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}) => {
+  return (
+    <div className="movie-card">
+      <span className="rate">{vote_average}</span>
+      <img className="movie-image" src={IMAGE_API + poster_path} alt={name} />
+      <h2 className="movie-title">{title ? title : name}</h2>
+      <div className="movie-footer">
+        <h3 className="movie-type">{media_type}</h3>
+        <h3 className="movie-date">
+          {release_date ? release_date : "2022 - 6 - 2"}
```
Comment (faouzioudouh) [inline src/Componnets/movie-card/MovieCard.jsx:22]:
```suggestion
          {release_date ?? "2022 - 6 - 2"}
```

File: src/Componnets/movie-card/MovieCard.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  title,
+  vote_average,
+  media_type,
+  release_date,
+}) => {
+  return (
+    <div className="movie-card">
+      <span className="rate">{vote_average}</span>
+      <img className="movie-image" src={IMAGE_API + poster_path} alt={name} />
+      <h2 className="movie-title">{title ? title : name}</h2>
```
Comment (faouzioudouh) [inline src/Componnets/movie-card/MovieCard.jsx:18]:
```suggestion
      <h2 className="movie-title">{title ?? name}</h2>
```

File: src/Componnets/movie-card/MovieCard.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  poster_path,
+  title,
+  vote_average,
+  media_type,
+  release_date,
+}) => {
+  return (
+    <div className="movie-card">
+      <span className="rate">{vote_average}</span>
+      <img className="movie-image" src={IMAGE_API + poster_path} alt={name} />
```
Comment (faouzioudouh) [inline src/Componnets/movie-card/MovieCard.jsx:17]: you can also create a component for image, so you don't have to always add the `IMGE_API` prefix

Comment (younessoudouh) [reply src/Componnets/movie-card/MovieCard.jsx:17]: ok got you

File: src/Componnets/movie-card/MovieCard.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Poster } from "../poster/Poster";
+import "./MovieCard.css";
+
+export const MovieCard = ({
+  name,
+  poster_path,
```
Comment (faouzioudouh) [inline src/Componnets/movie-card/MovieCard.jsx:7]: please use camelCase naming convention 
```suggestion
  posterPath,
```

File: src/Componnets/movie-card/MovieCard.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  title,
+  vote_average,
+  media_type,
+  release_date,
```
Comment (faouzioudouh) [inline src/Componnets/movie-card/MovieCard.jsx:11]:
```suggestion
  voteAverage,
  mediaType,
  releaseDate,
```

File: src/Componnets/movie-card/MovieCard.jsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+}) => {
+  return (
+    <div className="movie-card">
+      <span className="rate">{vote_average}</span>
+      <Poster name={name} poster_path={poster_path} />
```
Comment (faouzioudouh) [inline src/Componnets/movie-card/MovieCard.jsx:16]:
```suggestion
      <Poster name={name} src={poster_path} />
```

File: src/Componnets/poster/Poster.jsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const Poster = ({ poster_path, name }) => {
+  const POSTER__URL =
+    (poster_path && "https://image.tmdb.org/t/p/w300" + poster_path) ||
+    "https://www.movienewz.com/img/films/poster-holder.jpg";
```
Comment (faouzioudouh) [inline src/Componnets/poster/Poster.jsx:6]: 🔥

**Review by faouzioudouh — APPROVED** [review]

**Pull request merged**